### ISSUE TITLE
No options under 'Collection' in create new material samples; can't split

### ISSUE THREAD

**Issue by Dieter-Kahl** [AAFC-BICoE/dina-planning#342]
Do options need to be created by a Super User again? Are options still stored but not visible to a lowly User like me? Please excuse my inexperience :)

![image](https://user-images.githubusercontent.com/72175262/193685967-1b5783c3-e742-475c-9b04-e9198b895db8.png)

**Comment by cgendreau** [comment]
yes, collections need to be created by super-user/admin. Ron created CPVC so it should be good now.
